crafty.py

```diff
@@ -248,7 +248,7 @@ def send_kill_command():
         # save a file in app/config/new_install so we know this is a new install
         helper.make_new_install_file()
 
-        admin_pass = helper.random_string_generator()
+        admin_pass = "Admin" #! Password has been set as Admin for the time being
```

Comment:
run this command to drop this from this pr

```
git checkout origin master path/to/your/copy/of/crafty.py
```

Reply:
This will drop the change from the PR. Not a good idea to keep a copy of the code on master that has such a login